src/extension.ts

```diff
@@ -436,6 +486,16 @@ function launchMetals(
         );
       });
     });
+    if (features.treeViewProvider) {
+      const packageJson = JSON.parse(
+        fs.readFileSync(
+          path.join(context.extensionPath, "package.json"),
+          "utf8"
+        )
+      );
```

Comment:
`require("./package.json")` should work, in theory.

Reply:
> in theory

I thought so as well but I wasted probably 30 minutes trying to get it working and eventually gave up and went this route.

I did a quick retry  and it seems to be reading a very old version of package.json

<img width="534" alt="Screenshot 2019-06-26 at 19 49 01" src="https://user-images.githubusercontent.com/1408093/60202734-8605b180-984b-11e9-9847-afb22b15c781.png">

Running `yarn install` didn't fix the problem.

I left a comment explaining.